docs/guides/bindsnoop.md

```diff
@@ -27,3 +27,6 @@ Tracing binds ... Hit Ctrl-C to end
 NODE             NAMESPACE        PODNAME          CONTAINERNAME        PID COMM         PROT ADDR            PORT   OPTS IF
 ip-10-0-30-247   default          nginx-app        nginx-app         186667 nginx        UNKN 0.0.0.0            80 ...R.  0
 ```
+
+**Note:** It's possible that this example doesn't work if you're using the
+podinformer [hook](../install.md#hook-mode).
```

Comment:
Nit: I would write directly here a short sentence explaining the reason why it could not work.

Reply:
Done.